doc/source/ray-core/examples/batch_prediction.ipynb

```diff
@@ -0,0 +1,347 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Batch Prediction\n",
+    "\n",
+    "```{tip}\n",
+    "We strongly recommend using [Ray Datasets](https://docs.ray.io/en/latest/data/examples/nyc_taxi_basic_processing.html#parallel-batch-inference) or {doc}`AIR predictors </ray-air/predictors>` to develop batch prediction, which will enable you to build it faster and more easily, and get the built-in benefits like auto-scaling actor pool. If you think your use case cannot be supported by Ray Datasets or AIR, please file an issue in [Ray GitHub](https://github.com/ray-project/ray/issues) to get advice before you proceed. \n",
```

Comment:
I would remove the `to get advice before you proceed` bit, the language is a bit too strong.

Consider something like
```
If you think your use case cannot be supported by Ray Datasets or AIR, we'd love to get your feedback e.g. through a [Ray GitHub issue](https://github.com/ray-project/ray/issues).
```